src/lib/containers/QueryWrapperMenu.tsx

```diff
@@ -182,14 +183,21 @@ export default class QueryWrapperMenu extends React.Component<
         activeMenuIndices,
         accordionGroupIndex: accordionIndexIn,
       })
-      const facetName = this.props.menuConfig?.[menuIndexIn].facet
-      DeepLinkingUtils.updateUrlWithNewSearchParam(
-        'menuIndex',
-        undefined,
-        accordionIndexIn.toString(),
-      )
-      if (facetName) {
-        DeepLinkingUtils.updateUrlWithNewSearchParam('facet', undefined, facetName)
+
```

Comment:
I found one small bug - `shouldDeepLink` need's to get forwarded to QueryWrapper on line 375, then this can get merged!